docs/src/components/PlaygroundCodeEditor.tsx

```diff
@@ -0,0 +1,15 @@
+import clsx from 'clsx';
+import styles from '../pages/playground.module.css';
+
+function PlaygroundCodeEditor({ onChange, textAreaValue }) {
```

Comment:
- All the props should be typed (this applies to all the other components too)
- `textAreaValue` is a bad name. Think about level of abstraction you are at (PlaygroundCodeEditor) and think what this value represents in this context. Textarea is internal implementation detail of this component.
- You will probably want to pass `onChange(scene: object)` to PlaygroundCodeEditor and create wrapper function here that you pass to textarea